FTheoryToolsInOSCAR.ipynb

```diff
@@ -860,32 +1124,1130 @@
+    "triangulations of all facets of the polytope [HT17](@cite). A first indication for\n",
+    "the complexity of this triangulation task is the maximum number of lattice points\n",
+    "in a facet of the polytope in question. The following method returns this maximal number of\n",
+    "lattice points:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 55,
+   "id": "9c48f30d-601f-4337-88fd-b64bb14b6e8d",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "16"
+      ]
+     },
+     "execution_count": 55,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "max_lattice_pts_in_facet(qsm_model)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "cdcf8333-e3bf-40ec-923b-64a39e8a4d5c",
+   "metadata": {},
+   "source": [
+    "The following method returns an estimate for the number of full, regular, star triangulations\n",
+    "of the 3-dimensional reflexive polytope, those triangulations define the possible base\n",
+    "spaces of the F-theory QSM in question:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 56,
+   "id": "090d49b9-87f3-4959-b663-42c364962af6",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "212533333333"
+      ]
+     },
+     "execution_count": 56,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "estimated_number_of_triangulations(qsm_model)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "4138f688-1ea7-47e8-803a-1c6be32f2881",
+   "metadata": {},
+   "source": [
+    "Beyond the polytope and its triangulations, a number of other integers are of key importance, for example, Hodge numbers are also supported in the database."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "35917b76-4f39-47dd-9d6a-b9655a514bd4",
+   "metadata": {},
+   "source": [
+    "## 3.B The $C_{i}$ Curves"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "ab265d31-2d56-436e-b475-52b58fd31aad",
+   "metadata": {},
+   "source": [
+    "More recently, a research program estimated the exact massless spectra of the F-theory QSMs\n",
+    "(cf. [Bie24](@cite)). These studies require yet more information about the F-theory QSM geometries,\n",
+    "which are supported by the database.\n",
+    "\n",
+    "First, recall that (currently), the base of an F-theory QSM is a 3-dimensional toric variety $B3$.\n",
+    "Let $s \\in H^0(B3, \\bar{K}_{B3}^{\\otimes i})$, then $V(s)$ is a K3-surface. Moreover, let $x_{i}$ be the coordinates\n",
+    "of the Cox ring of $B3$. Then $V(x_{i})$ is a divisor in $B3$. Consequently, $C_{i} = V(x_{i}) \\cap V(s)$\n",
+    "is a divisor in the K3-surface $V(s)$. For the root bundle counting program, these curves $C_{i}$ are\n",
+    "of ample importance (cf. [Bie24](@cite)). The following information on these curves is supported:"
```

Comment:
Out of curiosity - jupyter notebook is able to render LaTeX code. Does it render those formulas correctly?

(I have this text to the main OSCAR documentation in a haze, so that this PR made it into the OSCAR 1.1 release. So they do not (yet) work in the main OSCAR documentation. Or so I believe.)

Reply:
They rendered correctly for me

Reply:
Wonderful.